### PR TITLE
Building weather app

### PR DESCRIPTION
**Pull request by Salvador-ON**
This is the weather -app using Javascript

task Dones
- [x] Build Static Html.
- [X] Build Program Logic.
- [X] Request Celsius information.
- [X] Request Farenheit information. 
- [X] Display the information.
- [X] Separate Dom Manipulation and Logic.
- [X] Use webpack.
- [X] Use ES6.

### CODE REVIEW

**Comment by Salvador-ON** [comment]
stickler-ci  is Successful in 3s — No lint errors found

**Comment by Salvador-ON** [comment]
GitGuardian Security Checks is showing  an error because the OpenWeatherMap Token is saved in the repo for the live version

File: dist/index.html [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+  <div class="full-cont">
+    <nav class="navbar navbar-dark bg-blue">
+      <a class="navbar-brand" href="#">
+        <img src="./assets/media/logo.png" width="30" height="30" class="d-inline-block align-top" alt="">
+        How is the Weather?
+      </a>
+
+      <form class="form-inline">
+        <input class="form-control mr-sm-2" type="search" placeholder="City" aria-label="Search" id="cityInput">
+        <button class="btn btn btn-light my-2 my-sm-0 mr-1" type="button" id="celsius">Celsius</button>
```
Comment (bellom) [inline dist/index.html:24]: Opps, I guess you miss some important information
 Your website looks really nice but we need to follow the necessary instructions given on `Odin` page as you know all Microverse students follow it.
so, we need to use `toggle` not as a form button`


You need to add toggle control to determine if the result will be shown in Celsius or Fahrenheit (According to the Odin instructions, screenshot below;)

image

https://www.theodinproject.com/courses/javascript/lessons/weather-app

File: src/index.js [hunk 2]
```diff
@@ -0,0 +1,42 @@
+import * as domMan from './domManipulation';
+
+const checkWeather = (system, city) => {
+  fetch(`https://api.openweathermap.org/data/2.5/weather?q=${city}&units=${system}&appid=0747aa2b81c66ff632767f1576af3d12`, { mode: 'cors' })
```
Comment (bellom) [inline src/index.js]: Opps, I guess you miss some important information again 😄 
One of the reasons for this project is to tests the knowledge of the student about asynchronous communication with promises or async/await and when to use them.

`Kindly make use of async-and-await`

Helpful Link: https://www.theodinproject.com/courses/javascript/lessons/async-and-await

**Review by bellom — CHANGES_REQUESTED** [review]
## Hi @Salvador-ON  :clap: 

### Awesome job so far!! :+1: 

#### You have done wonderfully well, you passed `almost` all the checklist for this project :+1: just a few issues left to work on............ :slightly_smiling_face: 
 
**Kindly check out my comments below;**

**Kindly re-submit for review after making all those changes :slightly_smiling_face: Thumps -up await you.....**

**Best,**

**Oluwaseun Bello**
**_Slack Handle - @sean bellom_**

File: dist/index.html (hunk 1 above)
Comment (Salvador-ON) [reply dist/index.html:24]: Ok thanks i will fix it

File: src/index.js (hunk 2 above)
Comment (Salvador-ON) [reply src/index.js]: ok thanks i'll fix it

**Comment by Salvador-ON** [comment]
i add the async and wait for my  promise function and add the toggle option to change from C to F and inverse

**Comment by Salvador-ON** [comment]
image

**Review by bellom — APPROVED** [review]
## Hi @Salvador-ON  :clap: 

**Great work making all the necessary corrections from the previous review.**

## Congratulation :+1:  
_Go ahead and merge to dev and don't forget to submit project completion form.
Good luck in the next project :slightly_smiling_face:_ 

**Best,**

**Oluwaseun Bello**
**_Slack Handle - @sean bellom_**

**Comment by Salvador-ON** [comment]
Thanks I'll proceed to merge

**Pull request merged**